src/app/components/templates/DefaultPages/Become-A-Shopper.tsx

```diff
@@ -31,25 +31,25 @@ const BecomeAShopper:React.FC =()=> {
-sites still in their infancy. Various versions have evolved over the years, sometimes by accident, sometimes on purpose (injected humour and the like).t is a long established fact that a reader will be distracted by the readable content of a page when looking at its layout. The point of using Lorem Ipsum is that it has a more-or-less normal distribution of letters, as opposed to using &apos;Content here, content here&apos;, making it look like readable English. Many desktop publishing packages and web page editors now use Lorem Ipsum as their default model text, and a search for &apos;lorem ipsum&apos; will uncover many web sites still in their infancy. Various versions have evolved over the years, sometimes by accident, sometimes on purpose (injected humour and the like).t is a long established fact that a reader will be distracted by the readable content of a page when looking at its layout.
+sites still in their infancy. Various versions have evolved over the years, sometimes by accident, sometimes on purpose (injected humour and the like).t is a long established fact that a reader will be distracted by the readable content of a page when looking at its layout. The point of using Lorem Ipsum is that it has a more-or-less normal distribution of letters, as opposed to using 'Content here, content here', making it look like readable English. Many desktop publishing packages and web page editors now use Lorem Ipsum as their default model text, and a search for 'lorem ipsum' will uncover many web sites still in their infancy. Various versions have evolved over the years, sometimes by accident, sometimes on purpose (injected humour and the like).t is a long established fact that a reader will be distracted by the readable content of a page when looking at its layout.
 
 The point of using Lorem Ipsum is that it has a more-or-less normal distribution of letters, as opposed to
- using &apos;Content here, content here&apos;, making it look like readable English. Many desktop publishing packages
-  and web page editors now use Lorem Ipsum as their default model text, and a search for &apos;lorem ipsum&apos; will 
+ using 'Content here, content here', making it look like readable English. Many desktop publishing packages
+  and web page editors now use Lorem Ipsum as their default model text, and a search for 'lorem ipsum' will 
   uncover many web sites still in their infancy. Various versions have evolved over the years, sometimes by 
   accident, sometimes on purpose (injected humour and the like).t is a long established fact that a reader will
    be distracted by the readable content of a page when looking at its layout. The point of using Lorem Ipsum is 
-   that it has a more-or-less normal distribution of letters, as opposed to using &apos;Content here, content here&apos;,
+   that it has a more-or-less normal distribution of letters, as opposed to using 'Content here, content here',
```

Comment:
I made changes to the way single quotes, double quotes and apostrophes are rendered. It wasn't getting compiled when deploying, so meaning replace `' `with `&apos;`  in this case here